src/usePromise.ts

```diff
@@ -26,6 +26,11 @@ export type PromiseOptions<T extends FunctionReturningPromise | FunctionReturnin
    * wait util you have all the arguments ready to execute the function.
    */
   execute?: boolean;
+  /**
+   * Custom error title for the generic failure toast.
+   * This will replace "Failed to fetch latest data".
+   */
+  errorTitle?: string;
```

Comment:
How about `failureToast?: (error: any) => ({title?: string, message?: string)}`? This would give even more flexibility?